winml/lib/Api.Ort/UniqueOrtPtr.h

```diff
@@ -7,15 +7,15 @@
 
 #include "adapter/winml_adapter_c_api.h"
 
```

Comment:
Surrounding this block with...

```
// clang-format off
nicely readable formatted stuff
/// clang-format on
```

...would preserve the visually scannable types.